docs/support/tutorials/blender-tutorial.md

```diff
@@ -0,0 +1,52 @@
+# Blender 
+
+These tutorial videos teach you how to use [Blender software](https://www.blender.org/) for scientific visualisation. 
+The series focuses on features useful for scientists, and correspond to one full day hands-on course.
+Note that the videos have caption texts, which you can switch on in YouTube (the "CC" button in the right bottom corner of the video).
+These videos were created by Jyrki Hokkanen in late 2020.
+Materials for hands-on exercises can be downloaded from Allas (the links are provided in the sections below).
+
+* [More about scientific visualisation at CSC](https://research.csc.fi/visualization)
+
+[TOC]
```

Comment:
```suggestion
```

Reply:
the TOC is in the right panel, and the texts between embedded iframes are small, so perhaps the TOC here is redundant?

Reply:
I agree!